cpp/environment/LocalDescriptors.cc

```diff
@@ -30,7 +30,7 @@ void LocalDescriptors::compute(const box::Box& box, const freud::locality::Neigh
                                const vec3<float>* r, unsigned int Np, const quat<float>* q_ref,
```

Comment:
I've written a test for this, let me push it and then we can adapt this code to the new method.

Reply:
The LocalDescriptors computation assumes knowledge of the number of bonds, so it will remain a computation on a NeighborList.